docs/source/cloud-setup/cloud-permissions/gcp.rst

```diff
@@ -0,0 +1,152 @@
+.. _cloud-permissions-gcp:
+
+GCP
+=============
+
```

Comment:
Generally, the administrator can choose among three "levels" of permissions, from the most permissive and least setup effort, to the least permissive and more setup effort:

- **Default**: no setup, give users Owner-level permissions (i.e., you do not need to follow the instructions here)
- **Medium-level**: easy setup, with a medium set of permissions
- **Minimal**: more setup, with the minimal set of permissions

(Please check if correct)